### ISSUE TITLE
Expose links to funding and tenders portal for matched beneficiaries in UI

### ISSUE THREAD

**Issue by Dgojsic** [ec-doris/kohesio-frontend#1007]
All matched beneficiaries with Property P998 should have the link there accessible in UI. Should appear on the left side in the "more info" section with text "Read about the Beneficiary's HORIZON-funded projects"

**Comment by madewild** [comment]
Sorry already created on backend side https://github.com/ec-doris/kohesio-backend/issues/269

We can follow-up here when implemented in the backend

**Comment by madewild** [comment]
@ntorbev the PIC number + URL is now exposed on https://kohesio.development.ec.europa.eu/api/beneficiaries/Q262833?language=en&id=https://linkedopendata.eu/entity/Q262833

@raphdom we can put "Read more on ..." or similar to avoid new translations?

**Comment by raphdom** [comment]
Yes, we can duplicate and reuse the "Read more on" from page.project-detail.link.readFurtherKeepEU label
